lib/apollo-federation/field_set_serializer.rb

```diff
@@ -6,29 +6,27 @@ module ApolloFederation
   module FieldSetSerializer
     extend self
 
-    def serialize(fields)
+    def serialize(fields, camelize: true)
       case fields
       when Hash
         fields.map do |field, nested_selections|
-          "#{camelize(field)} { #{serialize(nested_selections)} }"
+          "#{camelize(field, camelize)} { #{serialize(nested_selections, camelize: camelize)} }"
         end.join(' ')
       when Array
         fields.map do |field|
-          serialize(field)
+          serialize(field, camelize: camelize)
         end.join(' ')
-      when String
-        fields
-      when Symbol
-        camelize(fields)
+      when Symbol, String
+        camelize(fields, camelize)
       else
         raise ArgumentError, "Unexpected field set type: #{fields.class}"
       end
     end
 
     private
 
-    def camelize(field)
-      GraphQL::Schema::Member::BuildType.camelize(field.to_s)
+    def camelize(field, camelize)
+      camelize ? GraphQL::Schema::Member::BuildType.camelize(field.to_s) : field.to_s
```

Comment:
I tested the behavior of this helper. 

Camel_Me => camelMe 
camel_me => camelMe
camelMe => camelMe 
camelme => camelme 

I think this is the behavior we want. All it does is to lower camelcase snaked case stuff. If you had a slightly different camelisation scheme, it doesn't touch it. :+1: